### ISSUE TITLE
 Default currency change wont reflect in customer portal.

### ISSUE THREAD

**Issue by NetworkWebMasters** [InvoiceShelf/InvoiceShelf#142]
I set default currency INR Rupees while installing crater invoice, now I upgraded this to invoiceshelf.

i added estimate for customer and logged in customer panel.

Steps:- Login to customer panel -> Sidebar navigation menu -> Estimates -> Total Column -> Amount shows in $

Wrong currency in column "Total". It must be Rs. in that column. Please check this issue.

and also please add set different currency in different company account.

**Comment by gdarko** [comment]
The currency for specific customer can be set when editing the customer itself. 

There is a "Primary Currency" field.

**Comment by gdarko** [comment]
Closing for inactivity

**Comment by mchev** [comment]
> The currency for specific customer can be set when editing the customer itself.

`/admin/customers/1/edit`

![Capture d’écran 2024-08-29 à 08 33 28](https://github.com/user-attachments/assets/32a89d47-9d1d-40a7-83b7-7f9843c70702)

**Comment by NetworkWebMasters** [comment]
Any update on this?

**Comment by rihards-simanovics** [comment]
> Any update on this?

No, as of yet, sorry. This is on our radar. @mchev; @gdarko, could we look into this? It seems like a bug.

**Comment by NetworkWebMasters** [comment]
Any update on this?

**Comment by rihards-simanovics** [comment]
> Any update on this?

Unfortunately no. PRs are welcome!

**Comment by gdarko** [comment]
Fixed in v3.0 — the customer portal bootstrap now returns the company's default currency. When a customer has no currency assigned, the portal falls back to the company currency. Thank you for reporting!
